privacyidea/api/lib/prepolicy.py

```diff
@@ -1161,6 +1161,16 @@ def check_base_action(request=None, action=None, anonymous=False):
             realm = get_realms_of_token(request.view_args.get("serial"),
                                         only_first_realm=True)
 
+        # if the resolver was not given in the request,
+        # get the resolver by the serial either from request or from view_args
+        if not resolver:
+            if params.get("serial"):
+                user_object = get_token_owner(params.get("serial"))
+                resolver = user_object.resolver if user_object.resolver else None
+            elif request.view_args and request.view_args.get("serial"):
```

Comment:
params contains all_data, so it would also contain the view_args. This branch of the if-statement would never be touched to my understanding.

Reply:
OK, I understand the serial being part of the URL.